README.md

```diff
@@ -81,6 +82,66 @@ Converts components from synthetic shadow to native shadow.
 
 The only transformation it currently applies is to add the `static shadowSupportMode` property. Any other discrepancies between native shadow and synthetic shadow will have to be handled manually.
 
+### HTML Template Cleanup
+
+#### Usage
+
+```sh
+lwc-codemod html-template-cleanup <path>
+```
+
+#### Summary
+
+Fixes several HTML warnings/errors generated by the `@lwc/template-compiler`
+
+##### Multiple `if:true`/`if:false` Directives on the Same Element
+
+Removes excessive `if:true` and `if:false` attributes that appear on the same element. 
+
+Only the first `if:true`/`if:false` is processed in LWC, this mod will remove all other instances.
```

Comment:
```suggestion
Only the first `if:true`/`if:false` is processed in LWC. This mod will remove all other instances, since they would have no effect.
```